prdoc/6778.prdoc

```diff
@@ -0,0 +1,47 @@
+title: Added fallback_max_weight to Transact for sending messages to V4 chains
+doc:
+- audience: Runtime Dev
+  description: |-
+    Removing the `require_weight_at_most` parameter in V5 Transact introduced a problem when converting a message from V5 to V4 to send to chains that didn't upgrade yet.
+    The local chain doesn't know how to decode calls for remote chains so it can't automatically populate `require_weight_at_most` required by V4 Transact.
+    To fix this, XCM v5 Transact now also takes a `fallback_max_weight: Option<Weight>` parameter.
+    This can be set to `None` if the instruction is not meant to be sent to chains running XCM versions lower than V5.
+    If set to `Some(weight)`, a subsequent conversion to V4 will result in `Transact { require_weight_at_most: weight, .. }`.
+    The plan is to remove this workaround in V6 since there will be a good conversion path from V6 to V5.
```

Comment:
Do we have already have a list of things we want to have in v6? It'd be good to add this one straight away so that we don't forget about it.